ADAL/src/ADAuthenticationContext+Internal.m

```diff
@@ -83,8 +83,16 @@ + (ADAuthenticationError*)errorFromDictionary:(NSDictionary*)dictionary
         NSUUID* correlationId = [dictionary objectForKey:OAUTH2_CORRELATION_ID_RESPONSE] ?
                                 [[NSUUID alloc] initWithUUIDString:[dictionary objectForKey:OAUTH2_CORRELATION_ID_RESPONSE]]:
                                 nil;
-        SAFE_ARC_AUTORELEASE(correlationId);
-        return [ADAuthenticationError OAuthServerError:serverOAuth2Error description:errorDetails code:errorCode correlationId:correlationId];
+        SAFE_ARC_AUTORELEASE(correlationId);        
+        NSString *subcode = [dictionary objectForKey:OAUTH2_ERROR_SUBCODE];
```

Comment:
The feedback we got from the server team was that this is not guaranteed beyond one narrow scenario. The proper fix here it to pass "haschrome=1" as an extra query parameter so the "Back" button doesn't show up in the webview.